README.md

```diff
@@ -1,31 +1,19 @@
-# AWS IoT Device SDK C v4.0.0
+# AWS IoT Device SDK C with Fleet Provisioning<sup>BETA</sup> support
 
 **[Link to API documentation](https://docs.aws.amazon.com/freertos/latest/lib-ref/c-sdk/main/index.html)**
 
-[![Build Status](https://travis-ci.org/aws/aws-iot-device-sdk-embedded-C.svg?branch=v4_beta)](https://travis-ci.org/aws/aws-iot-device-sdk-embedded-C)
-[![Coverage Status](https://coveralls.io/repos/github/aws/aws-iot-device-sdk-embedded-C/badge.svg?branch=v4_beta)](https://coveralls.io/github/aws/aws-iot-device-sdk-embedded-C?branch=v4_beta)
+[![Build Status](https://travis-ci.org/aws/aws-iot-device-sdk-embedded-C.svg?branch=FleetProvisioning_beta)](https://travis-ci.org/aws/aws-iot-device-sdk-embedded-C)
+[![Coverage Status](https://coveralls.io/repos/github/aws/aws-iot-device-sdk-embedded-C/badge.svg?branch=FleetProvisioning_beta)](https://coveralls.io/github/aws/aws-iot-device-sdk-embedded-C?branch=FleetProvisioning_beta)
```

Comment:
is coverage supported yet? If so is it a good plan to keep it as a readme message